index.html

```diff
@@ -3587,13 +3587,18 @@ <h5><code>subscribeevent</code></h5>
 	  <span class="rfc2119-assertion" 
             id="http-sse-profile-protocol-binding-subscribeevent-6d">
             The <code>id</code> field SHOULD be set to a unique identifier for 
-            the event, for use when re-establishing a dropped connection.</span>  
-            See below.
+            the event, for use when re-establishing a dropped connection (see 
+            below).</span>
+    <span class="rfc2119-assertion" 
+            id="http-sse-profile-protocol-binding-subscribeevent-6e">
+            It is RECOMMENDED that the identifier is a timestamp 
+            representing the time at which the event ocurred, in the 
+            &quot;date-time&quot; format specified by [[RFC3339]].</span>
```

Comment:
I like this recommendation. Should we also provide a fallback mechanism, e.g. using numbers that get incremented when no timestamp is available?

Reply:
I'm not opposed to defining a fallback, but what if the device doesn't have non-volatile storage and can't keep track of an incrementing number across reboots? How many layers of fallback should we define..?